content/en/BestPractices/defend-against-ssrf.md

```diff
@@ -0,0 +1,257 @@
+# Defend Against Server-Side Request Forgery
+## Description 
+
+[Server-Side Request Forgery (SSRF)](https://owasp.org/Top10/A10_2021-Server-Side_Request_Forgery_%28SSRF%29/) is a server-side security vulnerability that allows a malicious actor (attacker) to make arbitrary requests from the application's server. 
+
```

Comment:
I'd suggest putting the CapitalOne paragraph here, as part of our intro, so that we can keep the "Impacts" paragraphs with the list of possible impacts.